src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -915,6 +916,23 @@ public Inventory get(final InventoryClickEvent e) {
 				return e.getClickedInventory();
 			}
 		}, 0);
+		//BlockFertilizeEvent
+		if(Skript.classExists("org.bukkit.event.block.BlockFertilizeEvent")) {
+			EventValues.registerEventValue(BlockFertilizeEvent.class, Block.class, new Getter<Block, BlockFertilizeEvent>() {
```

Comment:
Just out of curiosity, did you test without these getters?
If im not mistaken Skript automatically gets block and player from events with these methods, I could be wrong.

Reply:
Oh I didn't know that, I'll test

Reply:
It works with block but not with player 🤔

Reply:
Ok good to know. I know that often they're wonky, but I that could be an issue with how we've got them registered elsewhere.